ribs/archives/_cvt_archive.py

```diff
@@ -260,12 +264,30 @@ def index_of(self, measures_batch):
             _, indices = self._centroid_kd_tree.query(measures_batch)
             return indices.astype(np.int32)
 
```

Comment:
Can you put the stuff below this in an `else:` block? Should make it more readable. Lmk if pylint raises an error; I think it will be "no-return-else" or something

Reply:
Pylint raised an error when returning in an else block

Reply:
Can you add `no-else-return` to the list of disabled pylint messages here? https://github.com/icaros-usc/pyribs/blob/master/.pylintrc#L59 That'll turn off the warning. I also don't really like the `no-else-return` message anyway.

Reply:
After indenting the else blocks pylint throws a line-too-long error for the following line:

else:
                # Brute force distance calculation -- start by taking the difference
                
Should this be disabled as well?

Reply:
For that, you should break the comment into two lines so that it's no longer over 80 chars